tests/playwright/pages/wp-admin-page.ts

```diff
@@ -190,8 +190,9 @@ export default class WpAdminPage extends BasePage {
 	}
 
 	async closeBlockEditorPopupIfVisible() {
```

Comment:
is this relevant to this PR ?

Reply:
I think new wp-env is introducing a new changes. Add block icon is now in iframe, so I have to fix it